src/rest/remote-api.ts

```diff
@@ -87,15 +87,15 @@ export interface IRemoteAPI {
     getSshKey<T = che.ssh.SshPair>(service: string, name: string): Promise<T>;
     getAllSshKey<T = che.ssh.SshPair>(service: string): Promise<T[]>;
     deleteSshKey(service: string, name: string): Promise<void>;
-    getCurrentUser(): Promise<User>;
+    getCurrentUser(token?: string): Promise<User>;
```

Comment:
Yeah, I know there is no any doc in this interface but maybe it's time to add some?

Like 
```
/**
* Return the current authenticated user
* 
* params:
* token: optional token should be used for requesting CheAPI. 
*        In case it's missing - the default one will be used. <- not sure what mean here default, like machine token stored somewhere?
getCurrentUser(token?: string): Promise<User>;
```
@olexii4 @akurinnoy WDYT? Could you help to find a good place for such doc and formulate the better doc I did?

Reply:
Added docs for methods added by me

Reply:
@sleshchenko the proposal you did is clear and concise
@vinokurig thanks for adding docs!